Source/ProjectFirma.Web/Views/DNRUplandRegion/Detail.cshtml

```diff
@@ -118,6 +120,68 @@
         </div>
     </div>
 </div>
+<div class="row">
+    <div class="col-md-5">
+        <div class="panel panelFirma">
+            <div class="panel-heading panelTitle">
+                <h2>
+                    @ModalDialogFormHelper.MakeEditIconLink(ViewDataTyped.EditContactUrl, "Edit Contact", ViewDataTyped.UserHasRegionManagePermissions)
+                    @FieldDefinition.Contact.GetFieldDefinitionLabel()
+                </h2>
+            </div>
+            <div class="panel-body">
+                <div class="row">
+                    <div class="col-sm-3 fieldLabel text-right">
+                        Address
+                    </div>
+                    <div class="col-sm-9">
+                        @ViewDataTyped.DNRUplandRegion.RegionAddress
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-sm-3 fieldLabel text-right">
+                        City
+                    </div>
+                    <div class="col-sm-9">
+                        @ViewDataTyped.DNRUplandRegion.RegionCity
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-sm-3 fieldLabel text-right">
+                        State
+                    </div>
+                    <div class="col-sm-9">
+                        @ViewDataTyped.DNRUplandRegion.RegionState
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-sm-3 fieldLabel text-right">
+                        Zip
+                    </div>
+                    <div class="col-sm-9">
+                        @Html.Raw(ViewDataTyped.DNRUplandRegion.RegionZip)
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-xs-3 fieldLabel text-right">
+                        Phone
+                    </div>
+                    <div class="col-sm-9">
+                        @ViewDataTyped.DNRUplandRegion.Phone
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-sm-3 fieldLabel text-right">
+                        Email
+                    </div>
+                    <div class="col-sm-9">
+                        @Html.Raw(ViewDataTyped.DNRUplandRegion.Email)
```

Comment:
Remove the Html.Raw here. Shouldn't need to use that except for a few rare cases.